python/semantic_kernel/connectors/ai/google/google_ai/google_ai_prompt_execution_settings.py

```diff
@@ -38,7 +38,10 @@ class GoogleAIChatPromptExecutionSettings(GoogleAIPromptExecutionSettings):
     tools: Annotated[
         list[dict[str, Any]] | None,
         Field(
-            max_length=64,
+            # There is no official documentation on the maximum length of the tools list.
+            # Using the limit stated on the Vertex AI documentation:
+            # https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/function-calling#function-declarations
+            max_length=128,
```

Comment:
Should we remove the max_length constraint completely so we don't need to continuously play "catch up" and update it again when Google updates it on their end? I think it's okay for there to be a server-side failure. Saves us from having to keep track of the max length for all of our connectors going forward.